src/main/java/io/citrine/jcc/search/core/result/BaseAtomicSearchResult.java

```diff
@@ -0,0 +1,28 @@
+package io.citrine.jcc.search.core.result;
+
+/**
+ * Created by maxhutch on 2/13/17.
+ */
+public class BaseAtomicSearchResult<T> {
+
+
+    public T getResult() {
+        return result;
+    }
+
+    public void setResult(T result) {
+        this.result = result;
+    }
+
+    public boolean isSuccess() {
```

Comment:
Oh just caught this. Related to comment on the other PR, I think this could have a throttled state in addition to success and error.